bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/jsons/JsonMusicProvider.java

```diff
@@ -25,9 +25,9 @@
 @NonNullByDefault
 public class JsonMusicProvider {
     public @Nullable String displayName;
-    public @Nullable List<@Nullable Object> @Nullable [] supportedTriggers;
+    public @Nullable List<Object> @Nullable [] supportedTriggers;
```

Comment:
From my understanding this is not a workaround but reflects the null-values from the JSON response (also immediately below)